assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -188,17 +194,27 @@ angular.module('openshiftConsole')
           );
         };
 
-        DataService.get("templates", name, {namespace: (namespace || $scope.projectName)}).then(
-          function(template) {
-            $scope.template = template;
-            $scope.templateImages = imageItems(template);
-            $scope.hasParameters = $scope.template.parameters && $scope.template.parameters.length > 0;
-            $scope.templateUrl = template.metadata.selfLink;
-            template.labels = template.labels || {};
-          },
-          function() {
-            Navigate.toErrorPage("Cannot create from template: the specified template could not be retrieved.");
-          });
+        function setTemplateParams() {
+          $scope.templateImages = imageItems($scope.template);
+          $scope.hasParameters = !_.isEmpty($scope.template.parameters);
+          $scope.templateUrl = $scope.template.metadata.selfLink;
```

Comment:
Actually, searching the code, I don't think `hasParameters` or `templateUrl` are used anymore. Can you confirm and remove them if so?